packages/react-examples/src/react-charting/AreaChart/AreaChart.Basic.Example.tsx

```diff
@@ -156,9 +156,23 @@ export class AreaChartBasicExample extends React.Component<{}, IAreaChartBasicSt
     return (
       <>
         <label>change Width:</label>
-        <input type="range" value={this.state.width} min={200} max={1000} onChange={this._onWidthChange} />
+        <input
+          type="range"
+          value={this.state.width}
+          min={200}
+          max={1000}
+          aria-label="change Width"
```

Comment:
I'd suggest to rather assign id to `label` so we leverage default a11y browser behaviour instead of manually adding aria to follow the best practice which says "no aria is the best aria"